sdk/core/azure-core/tests/test_tracing_implementations.py

```diff
@@ -93,17 +75,20 @@ def test_no_span_but_in_trace(self):
     def test_span(self):
         with ContextHelper() as ctx:
             tracer = tracer_module.Tracer(sampler=AlwaysOnSampler())
-            wrapped_class = OpencensusSpanWrapper()
+            wrapped_class = OpenCensusSpan()
             child = wrapped_class.span()
             assert child.span_instance.name == "child_span"
-            assert child.tracer.span_context.trace_id == tracer.span_context.trace_id
```

Comment:
Looks like you're formatting to 80 columns max. We're using 120 (`black -l 120`).

Reply:
done